config/environments/integration.rb

```diff
@@ -0,0 +1,113 @@
+# Settings for integration testings.  A mix between develpment and production,
```

Comment:
Typo `develpment` => `development`

Reply:
Literally got it right on the next line... 😩